packages/core/test/apiHelper.test.ts

```diff
@@ -73,6 +79,62 @@ describe('test user agent', () => {
   );
 });
 
+describe('test error template', () => {
+  test.each([
+    [
+      'test error message with statusCode, response headers and response.body templates to be replaced',
+      'Global Error template 500: {$statusCode}, accept => {$response.header.content-type}, body => {$response.body}.',
```

Comment:
Missing $response.header cases
- invalid syntax i.e missing the `.` after `$response.header`
- points to a missing header
- points to a correct header w/ different casing i.e. `CONTENT-TYPE`, `content-type`, `Content-Type` should all points to content-type header case insensitively
- points to a correct header whose name has spaces i.e. `content type`